packages/core/test/assets_gen_integrations_test.dart

```diff
@@ -31,8 +33,10 @@ void main() {
       expect(integration.className, 'SvgGenImage');
       expect(integration.classInstantiate('assets/path'),
           'SvgGenImage(\'assets/path\')');
-      expect(integration.isSupport(AssetType('assets/path/dog.svg')), isTrue);
-      expect(integration.isSupport(AssetType('assets/path/dog.png')), isFalse);
+      expect(integration.isSupport(AssetType('assets/path/dog.svg', resPath)),
```

Comment:
We don't add any variables to the AssetType, so no changes here are needed either.